phive.xml

```diff
@@ -1,35 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Please revert that file.

Reply:
Oh, it didnt commit the new file location it was suggesting in its latest version
Fixed now